tests/Oryx.Tests.Common/ImageTestHelper.cs

```diff
@@ -494,12 +494,6 @@ private string GetFullBuildImage(string buildImageTag = null)
                 NodeConstants.NodeToolName,
                 NodeVersions.RuntimeVersions
             },
-            {
```

Comment:
As mentioned offline: we should be OK to make the below change to the [`ShellOutput`](https://github.com/microsoft/Oryx/blob/main/build/tools/SharedCodeGenerator/Outputs/ShellOutput.cs#L56) file rather than making any changes in this PR to `.cs` files:

`!c.Key.Equals("runtime-versions")` --> `!c.Key.Contains("runtime-versions")`

Reply:
Created different PR for this: https://github.com/microsoft/Oryx/pull/2156/files